docusaurus/docs/api-endpoints/api/user/projects/get-project.md

```diff
@@ -0,0 +1,7 @@
+# GET /api/user/projects/:id
+
+Get the project data by id.
+
+## Response
+
+200 on success, 401 if no valid "user-token" cookie is found, and 404 if no project with given id is found. The response body contains the JSON string saved in the database.
```

Comment:
and 404 if current user id does not match projects owner id (same on PUT request)

Reply:
this is fixed in a later pull request